sed/core/processor.py

```diff
@@ -1180,23 +1180,25 @@ def calibrate_delay_axis(
             else:
                 print(self._dataframe)
 
-    def add_jitter(self, cols: Sequence[str] = None):
+    def add_jitter(self, cols: List[str] = None, **kwds):
```

Comment:
why use a different behaviour for cols and amps? should we not add it as argument here with default to None?

Reply:
I thought that amps should not be a default argument, because we would not per default use it. I wanted to provide access to the other arguements of the function, in case they are needed, but they should not be needed per default, as jittering should only be required on digital values, and then the jittering should be done with amp=0.5